packages/coding-standard/config/symplify-rules.neon

```diff
@@ -214,9 +214,7 @@ services:
                 - 'Symplify\CodingStandard\Rules\ForbiddenParentClassRule'
                 - 'Symplify\CodingStandard\Rules\NoClassWithStaticMethodWithoutStaticNameRule'
                 - 'Symplify\CodingStandard\Rules\PreferredClassRule'
```

Comment:
Are you able to clean the rest of these rules too?

Reply:
implemented.